benchmarking/commons/hpo_main_local.py

```diff
@@ -10,57 +10,84 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-from typing import Optional, List, Callable, Dict, Any
-import numpy as np
 import itertools
-from tqdm import tqdm
 import logging
+from typing import Optional, Callable, Dict, Any
+
+import numpy as np
+from tqdm import tqdm
 
-from syne_tune.backend import LocalBackend
-from syne_tune.stopping_criterion import StoppingCriterion
-from syne_tune.tuner import Tuner
 from benchmarking.commons.baselines import MethodArguments, MethodDefinitions
 from benchmarking.commons.benchmark_definitions.common import RealBenchmarkDefinition
 from benchmarking.commons.hpo_main_common import (
-    parse_args as _parse_args,
     set_logging_level,
     get_metadata,
     ExtraArgsType,
-    MapExtraArgsType,
+    MapMethodArgsType,
     PostProcessingType,
+    ConfigDict,
+    DictStrKey,
     extra_metadata,
+    str2bool,
 )
 from benchmarking.commons.utils import get_master_random_seed, effective_random_seed
+from syne_tune.backend import LocalBackend
+from syne_tune.stopping_criterion import StoppingCriterion
+from syne_tune.tuner import Tuner
 
 logger = logging.getLogger(__name__)
 
 
 RealBenchmarkDefinitions = Callable[..., Dict[str, RealBenchmarkDefinition]]
+LOCAL_LOCAL_BENCHMARK_REQUIRED_PARAMETERS = [
```

Comment:
It was local (for local backend) and local (for not-remote launching). I can rename to `LOCAL_BACKEND_EXTRA_PARAMETERS` since I gues we can assume local launcher is the default so dosnt need indicating.

Reply:
Well, the same CL parameters are used in both cases.